nextflow_schema.json

```diff
@@ -10,7 +10,7 @@
             "type": "object",
             "fa_icon": "fas fa-terminal",
             "description": "Define where the pipeline should find input data and save output data.",
-            "required": ["input", "outdir"],
+            "required": ["input", "mcl_inflation", "outdir", "fasta_dir"],
```

Comment:
if there is an mcl inflation as an input argument, what are the processes that run on test data to then figure out the inflation score?

Reply:
There's a few - 
1. orthofinder_prep - formats files to that orthofinder can do its thing. Sadly creates a messy directory structure that introduces challenges later on. 
2. diamond blastp - similarity scores for mcl
3. orthofinder_mcl - does the clustering on each provided mcl inflation parameter
4. cogqc - summarizes the orthogroups (and pulls annotations for the complete dataset produced from the annotate_uniprot process)
5. select_inflation - sumarizes the cogqc output for each mcl parameter - that parameter is then used to cluster proteins for the full dataset into orthogroups.

Reply:
oh geeze I completely botched this question - I meant to ask WHY are there processes that run on test data to figure out inflation scores when inflation score is provided as a parameter to the pipeline

Reply:
Hahah, no worries! It's admittedly a bit weird and should be described in the README. 

Basically, I don't know for a given dataset what the best inflation parameter to use is. So, I provide several to conduct a parameter sweep using the "test" dataset. 

- So the orthofinder_mcl module runs for each inflation parameter, those results are summarized (cogeqc)... 
- those summaries are used to select the best inflation parameter (select_inflation). 
- Then that best inflation parameter is provided again to the orthofinder_mcl module, this time a singular value and run using the complete dataset).